src/app/core/abstractions/entity-parser.service.ts

```diff
@@ -20,4 +21,6 @@ export abstract class EntityParser {
   public abstract parseMediaAsset(media: unknown): Media
   public abstract parseMediaAssets(media: unknown): Media[]
   public abstract parseSystemInformation(systemInformation: unknown): SystemInformation
+  public abstract parseStatusMessage(statusMessages: unknown): StatusMessage
+  public abstract parseStatusMessages(statusMessages: unknown): StatusMessage[]
```

Comment:
I see your point. I'm also in doubt whether it belongs here or not. But I would like us to avoid having a new parser for every single entity in the system. (We would need to move Media, SystemInformation, ShowstyleVariant etc from here as well).